bindings/python/scripts/compilation_database.py

```diff
@@ -0,0 +1,40 @@
+import clang.cindex as clang
+
+
+class CompilationDatabase:
+    """
+    Build a compilation database from a given directory
+    """
+
+    def __init__(self, compilation_database_path):
+        self.compilation_database = clang.CompilationDatabase.fromDirectory(
+            buildDir=compilation_database_path
+        )
+
+    def get_compilation_arguments(self, filename=None):
```

Comment:
Filename required here? Shouldn't the db path in L9 solve this?

Reply:
Filename for the cpp file to be parsed, to get that file's compiler args from the compilation database. 
If none, it will return the compiler args for all the files in the compilation db.